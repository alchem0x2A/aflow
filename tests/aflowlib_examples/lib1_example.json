{"aurl":"aflowlib.duke.edu:AFLOWDATA/LIB1_RAW/Ac:PAW_GGA:11Apr2000/A1","auid":"aflow:075dbbb922a0e4d7","data_api":"aapi1.2","data_source":"aflowlib","loop":["bands","lock","magnetic","thermodynamics"],"code":"vasp.4.6.35","compound":"Ac1","prototype":"A1","nspecies":1,"natoms":1,"natoms_orig":1,"composition":[1],"density":8.35677,"density_orig":8.33248,"scintillation_attenuation_length":0.714705,"stoichiometry":[1],"species":["Ac"],"species_pp":["Ac"],"dft_type":["PAW_GGA"],"species_pp_version":["Ac:PAW_GGA:11Apr2000"],"species_pp_ZVAL":[11],"species_pp_AUID":["f04c8b0982efae26"],"valence_cell_iupac":3,"valence_cell_std":3,"volume_cell":45.1122,"volume_atom":45.1122,"volume_cell_orig":45.2437,"volume_atom_orig":45.2437,"pressure":0,"stress_tensor":[-0.23,-0,0,-0,-0.23,0,0,0,-0.23],"pressure_residual":-0.23,"Pulay_stress":0,"geometry":[3.995793,3.995793,3.995793,60,60,60],"geometry_orig":[3.999672,3.999672,3.999672,60,60,60],"energy_cell":-4.06438,"energy_atom":-4.06438,"energy_cutoff":237.9,"delta_electronic_energy_convergence":6.91737e-08,"delta_electronic_energy_threshold":1e-06,"kpoints_relax":[32,32,32],"kpoints_static":[31,31,31],"kpoints_bands_path":["\\Gamma-X","X-W","W-K","K-\\Gamma","\\Gamma-L","L-U","U-W","W-L","L-K","U-X"],"kpoints_bands_nkpts":128,"kpoints":[[32,32,32],[31,31,31],["\\Gamma-X","X-W","W-K","K-\\Gamma","\\Gamma-L","L-U","U-W","W-L","L-K","U-X"],128],"enthalpy_cell":-4.06438,"enthalpy_atom":-4.06438,"eentropy_cell":-0.00058382,"eentropy_atom":-0.00058382,"enthalpy_formation_cell":0,"enthalpy_formation_atom":0,"entropic_temperature":0,"PV_cell":0,"PV_atom":0,"spin_cell":0,"spin_atom":0,"spinD":[0],"spinF":0,"calculation_time":10640.6,"calculation_memory":1515.35,"calculation_cores":8,"nbondxx":[3.99579307],"sg":["Fm-3m #225","Fm-3m #225","Fm-3m #225"],"sg2":["Fm-3m #225","Fm-3m #225","Fm-3m #225"],"spacegroup_orig":225,"spacegroup_relax":225,"forces":[[0,0,-0]],"positions_cartesian":[[0,-0,-0]],"positions_fractional":[[-0,0,-0]],"Bravais_lattice_orig":"FCC","lattice_variation_orig":"FCC","lattice_system_orig":"cubic","Pearson_symbol_orig":"cF4","Bravais_lattice_relax":"FCC","lattice_variation_relax":"FCC","lattice_system_relax":"cubic","Pearson_symbol_relax":"cF4","crystal_family_orig":"cubic","crystal_system_orig":"cubic","crystal_class_orig":"hexoctahedral","point_group_Hermann_Mauguin_orig":"m-3m","point_group_Schoenflies_orig":"O_h","point_group_orbifold_orig":"*432","point_group_type_orig":"centrosymmetric","point_group_order_orig":48,"point_group_structure_orig":"2_x_symmetric","Bravais_lattice_lattice_type_orig":"FCC","Bravais_lattice_lattice_variation_type_orig":"FCC","Bravais_lattice_lattice_system_orig":"cubic","Bravais_superlattice_lattice_type_orig":"FCC","Bravais_superlattice_lattice_variation_type_orig":"FCC","Bravais_superlattice_lattice_system_orig":"cubic","Pearson_symbol_superlattice_orig":"cF4","reciprocal_geometry_orig":[1.924,1.924,1.924,109.47,109.47,109.47],"reciprocal_volume_cell_orig":5.48254,"reciprocal_lattice_type_orig":"BCC","reciprocal_lattice_variation_type_orig":"BCC","Wyckoff_letters_orig":[["a"]],"Wyckoff_multiplicities_orig":[[4]],"Wyckoff_site_symmetries_orig":[["m-3m"]],"crystal_family":"cubic","crystal_system":"cubic","crystal_class":"hexoctahedral","point_group_Hermann_Mauguin":"m-3m","point_group_Schoenflies":"O_h","point_group_orbifold":"*432","point_group_type":"centrosymmetric","point_group_order":48,"point_group_structure":"2_x_symmetric","Bravais_lattice_lattice_type":"FCC","Bravais_lattice_lattice_variation_type":"FCC","Bravais_lattice_lattice_system":"cubic","Bravais_superlattice_lattice_type":"FCC","Bravais_superlattice_lattice_variation_type":"FCC","Bravais_superlattice_lattice_system":"cubic","Pearson_symbol_superlattice":"cF4","reciprocal_geometry":[1.9259,1.9259,1.9259,109.47,109.47,109.47],"reciprocal_volume_cell":5.49852,"reciprocal_lattice_type":"BCC","reciprocal_lattice_variation_type":"BCC","Wyckoff_letters":[["a"]],"Wyckoff_multiplicities":[[4]],"Wyckoff_site_symmetries":[["m-3m"]],"anrl_label_orig":"A_cF4_225_a","anrl_parameter_list_orig":["a"],"anrl_parameter_values_orig":[5.6564],"anrl_label_relax":"A_cF4_225_a","anrl_parameter_list_relax":["a"],"anrl_parameter_values_relax":[5.6509],"files":["Ac.A1_banddos.png","Ac.A1_dos.png","Ac.A1_dos_Ac_1.png","Ac:PAW_GGA:11Apr2000.A1.cif","Ac:PAW_GGA:11Apr2000.A1_bandsdata.json.xz","Ac:PAW_GGA:11Apr2000.A1_corner.cif","Ac:PAW_GGA:11Apr2000.A1_dosdata.json.xz","Ac:PAW_GGA:11Apr2000.A1_sconv.cif","Ac:PAW_GGA:11Apr2000.A1_sconv_corner.cif","Ac:PAW_GGA:11Apr2000.A1_sprim.cif","Ac:PAW_GGA:11Apr2000.A1_sprim_corner.cif","Ac:PAW_GGA:11Apr2000.A1_structure_relax.json","Ac:PAW_GGA:11Apr2000.A1_structure_relax1.json","CONTCAR.relax","CONTCAR.relax.abinit","CONTCAR.relax.aims","CONTCAR.relax.qe","CONTCAR.relax.vasp","CONTCAR.relax1","DOSCAR.static.xz","EIGENVAL.bands.xz","INCAR.bands","KPOINTS.bands","KPOINTS.relax","KPOINTS.static","LOCK","OSZICAR.bands.xz","OSZICAR.static","OUTCAR.relax.xz","OUTCAR.static.xz","POSCAR.bands","POSCAR.orig","aflow.agroup.bands.json.xz","aflow.agroup.bands.out.xz","aflow.agroup.orig.json.xz","aflow.agroup.orig.out.xz","aflow.agroup.relax.json.xz","aflow.agroup.relax.out.xz","aflow.fgroup.bands.json.xz","aflow.fgroup.bands.out.xz","aflow.fgroup.orig.json.xz","aflow.fgroup.orig.out.xz","aflow.fgroup.relax.json.xz","aflow.fgroup.relax.out.xz","aflow.iatoms.bands.json.xz","aflow.iatoms.bands.out.xz","aflow.iatoms.orig.json.xz","aflow.iatoms.orig.out.xz","aflow.iatoms.relax.json.xz","aflow.iatoms.relax.out.xz","aflow.in","aflow.pgroup.bands.json.xz","aflow.pgroup.bands.out.xz","aflow.pgroup.orig.json.xz","aflow.pgroup.orig.out.xz","aflow.pgroup.relax.json.xz","aflow.pgroup.relax.out.xz","aflow.pgroup_xtal.bands.json.xz","aflow.pgroup_xtal.bands.out.xz","aflow.pgroup_xtal.orig.json.xz","aflow.pgroup_xtal.orig.out.xz","aflow.pgroup_xtal.relax.json.xz","aflow.pgroup_xtal.relax.out.xz","aflow.pgroupk.bands.json.xz","aflow.pgroupk.bands.out.xz","aflow.pgroupk.orig.json.xz","aflow.pgroupk.orig.out.xz","aflow.pgroupk.relax.json.xz","aflow.pgroupk.relax.out.xz","aflow.pgroupk_xtal.bands.json.xz","aflow.pgroupk_xtal.bands.out.xz","aflow.pgroupk_xtal.orig.json.xz","aflow.pgroupk_xtal.orig.out.xz","aflow.pgroupk_xtal.relax.json.xz","aflow.pgroupk_xtal.relax.out.xz","bz_.eps","bz_.png","edata.bands.json","edata.bands.out","edata.orig.json","edata.orig.out","edata.relax.json","edata.relax.out","plotbz.sh"],"aflow_version":"aflow30576","catalog":"LIB1","aflowlib_version":"3.2.2l","aflowlib_date":"20200507_11:23:35_GMT-5"}
