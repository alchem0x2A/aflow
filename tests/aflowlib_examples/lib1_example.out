aurl=aflowlib.duke.edu:AFLOWDATA/LIB1_RAW/Ba_pv:LDA:01Apr2000/A3 | auid=aflow:0241eab8a156fac2 | data_api=aapi1.2 | data_source=aflowlib | loop=bands,lock,magnetic,thermodynamics | code=vasp.4.6.35 | compound=Ba2 | prototype=A3 | nspecies=1 | natoms=2 | natoms_orig=2 | composition=2 | density=4.12893 | density_orig=4.12893 | scintillation_attenuation_length=2.64767 | stoichiometry=1 | species=Ba | species_pp=Ba | dft_type=LDA | species_pp_version=Ba:LDA:01Apr2000 | species_pp_ZVAL=8 | species_pp_AUID=b5d1dafcf2f8d798 | valence_cell_iupac=4 | valence_cell_std=4 | volume_cell=110.461 | volume_atom=55.2303 | volume_cell_orig=110.461 | volume_atom_orig=55.2303 | pressure=0 | stress_tensor=0,0,0,0,0,-0,0,-0,0.02 | pressure_residual=0.01 | Pulay_stress=0 | geometry=4.276211,4.276211,6.97523,90,90,120 | geometry_orig=4.276211,4.276211,6.97523,90,90,120 | energy_cell=-4.37368 | energy_atom=-2.18684 | energy_cutoff=141.7 | delta_electronic_energy_convergence=-1.19033e-07 | delta_electronic_energy_threshold=1e-06 | kpoints_relax=32,32,17 | kpoints_static=33,33,17 | kpoints_bands_path=\Gamma-M,M-K,K-\Gamma,\Gamma-A,A-L,L-H,H-A,L-M,K-H | kpoints_bands_nkpts=128 | kpoints=32,32,17;33,33,17;\Gamma-M,M-K,K-\Gamma,\Gamma-A,A-L,L-H,H-A,L-M,K-H;128 | enthalpy_cell=-4.37368 | enthalpy_atom=-2.18684 | eentropy_cell=0.00082331 | eentropy_atom=0.000411655 | enthalpy_formation_cell=0.017042 | enthalpy_formation_atom=0.008521 | entropic_temperature=0 | PV_cell=0 | PV_atom=0 | spin_cell=0 | spin_atom=0 | spinD=0,0 | spinF=0 | stoich=1.0000 | calculation_time=1522.19 | calculation_memory=662.48 | calculation_cores=16 | nbondxx=4.273 | sg=P6_{3}/mmc #194,P6_{3}/mmc #194,P6_{3}/mmc #194 | sg2=P6_{3}/mmc #194,P6_{3}/mmc #194,P6_{3}/mmc #194 | spacegroup_orig=194 | spacegroup_relax=194 | forces=0,0,0;0,0,0 | positions_cartesian=0,0,0;2.13811,-1.23444,3.48762 | positions_fractional=0,0,0;0.66666667,0.33333333,0.5 | Bravais_lattice_orig=HEX | lattice_variation_orig=HEX | lattice_system_orig=hexagonal | Pearson_symbol_orig=hP2 | Bravais_lattice_relax=HEX | lattice_variation_relax=HEX | lattice_system_relax=hexagonal | Pearson_symbol_relax=hP2 | crystal_family_orig=hexagonal | crystal_system_orig=hexagonal | crystal_class_orig=dihexagonal-dipyramidal | point_group_Hermann_Mauguin_orig=6/mmm | point_group_Schoenflies_orig=D_6h | point_group_orbifold_orig=*622 | point_group_type_orig=centrosymmetric | point_group_order_orig=24 | point_group_structure_orig=2_x_dihedral | Bravais_lattice_lattice_type_orig=HEX | Bravais_lattice_lattice_variation_type_orig=HEX | Bravais_lattice_lattice_system_orig=hexagonal | Bravais_superlattice_lattice_type_orig=HEX | Bravais_superlattice_lattice_variation_type_orig=HEX | Bravais_superlattice_lattice_system_orig=hexagonal | Pearson_symbol_superlattice_orig=hP2 | reciprocal_geometry_orig=1.6966,1.6966,0.90079,90,90,60 | reciprocal_volume_cell_orig=2.2456 | reciprocal_lattice_type_orig=HEX | reciprocal_lattice_variation_type_orig=HEX | Wyckoff_letters_orig=c | Wyckoff_multiplicities_orig=2 | Wyckoff_site_symmetries_orig=-6m2 | crystal_family=hexagonal | crystal_system=hexagonal | crystal_class=dihexagonal-dipyramidal | point_group_Hermann_Mauguin=6/mmm | point_group_Schoenflies=D_6h | point_group_orbifold=*622 | point_group_type=centrosymmetric | point_group_order=24 | point_group_structure=2_x_dihedral | Bravais_lattice_lattice_type=HEX | Bravais_lattice_lattice_variation_type=HEX | Bravais_lattice_lattice_system=hexagonal | Bravais_superlattice_lattice_type=HEX | Bravais_superlattice_lattice_variation_type=HEX | Bravais_superlattice_lattice_system=hexagonal | Pearson_symbol_superlattice=hP2 | reciprocal_geometry=1.6966,1.6966,0.90079,90,90,60 | reciprocal_volume_cell=2.2456 | reciprocal_lattice_type=HEX | reciprocal_lattice_variation_type=HEX | Wyckoff_letters=c | Wyckoff_multiplicities=2 | Wyckoff_site_symmetries=-6m2 | anrl_label_orig=A_hP2_194_c | anrl_parameter_list_orig=a,c/a | anrl_parameter_values_orig=4.2762,1.63117 | anrl_label_relax=A_hP2_194_c | anrl_parameter_list_relax=a,c/a | anrl_parameter_values_relax=4.2762,1.63117 | ael_stiffness_tensor= | ael_compliance_tensor= | files=Ba_pv.A3_banddos.png,Ba_pv.A3_dos.png,Ba_pv.A3_dos_Ba_1.png,Ba_pv:LDA:01Apr2000.A3.cif,Ba_pv:LDA:01Apr2000.A3_bandsdata.json.xz,Ba_pv:LDA:01Apr2000.A3_corner.cif,Ba_pv:LDA:01Apr2000.A3_dosdata.json.xz,Ba_pv:LDA:01Apr2000.A3_sconv.cif,Ba_pv:LDA:01Apr2000.A3_sconv_corner.cif,Ba_pv:LDA:01Apr2000.A3_sprim.cif,Ba_pv:LDA:01Apr2000.A3_sprim_corner.cif,Ba_pv:LDA:01Apr2000.A3_structure_relax.json,Ba_pv:LDA:01Apr2000.A3_structure_relax1.json,CONTCAR.relax,CONTCAR.relax.abinit,CONTCAR.relax.aims,CONTCAR.relax.qe,CONTCAR.relax.vasp,CONTCAR.relax1,DOSCAR.static.xz,EIGENVAL.bands.xz,INCAR.bands,KPOINTS.bands,KPOINTS.relax,KPOINTS.static,LOCK,OSZICAR.bands.xz,OSZICAR.static,OUTCAR.relax.xz,OUTCAR.static.xz,POSCAR.bands,POSCAR.orig,aflow.agroup.bands.json.xz,aflow.agroup.bands.out.xz,aflow.agroup.orig.json.xz,aflow.agroup.orig.out.xz,aflow.agroup.relax.json.xz,aflow.agroup.relax.out.xz,aflow.fgroup.bands.json.xz,aflow.fgroup.bands.out.xz,aflow.fgroup.orig.json.xz,aflow.fgroup.orig.out.xz,aflow.fgroup.relax.json.xz,aflow.fgroup.relax.out.xz,aflow.iatoms.bands.json.xz,aflow.iatoms.bands.out.xz,aflow.iatoms.orig.json.xz,aflow.iatoms.orig.out.xz,aflow.iatoms.relax.json.xz,aflow.iatoms.relax.out.xz,aflow.in,aflow.pgroup.bands.json.xz,aflow.pgroup.bands.out.xz,aflow.pgroup.orig.json.xz,aflow.pgroup.orig.out.xz,aflow.pgroup.relax.json.xz,aflow.pgroup.relax.out.xz,aflow.pgroup_xtal.bands.json.xz,aflow.pgroup_xtal.bands.out.xz,aflow.pgroup_xtal.orig.json.xz,aflow.pgroup_xtal.orig.out.xz,aflow.pgroup_xtal.relax.json.xz,aflow.pgroup_xtal.relax.out.xz,aflow.pgroupk.bands.json.xz,aflow.pgroupk.bands.out.xz,aflow.pgroupk.orig.json.xz,aflow.pgroupk.orig.out.xz,aflow.pgroupk.relax.json.xz,aflow.pgroupk.relax.out.xz,aflow.pgroupk_xtal.bands.json.xz,aflow.pgroupk_xtal.bands.out.xz,aflow.pgroupk_xtal.orig.json.xz,aflow.pgroupk_xtal.orig.out.xz,aflow.pgroupk_xtal.relax.json.xz,aflow.pgroupk_xtal.relax.out.xz,bz_.eps,bz_.png,edata.bands.json,edata.bands.out,edata.orig.json,edata.orig.out,edata.relax.json,edata.relax.out,plotbz.sh | aflow_version=aflow30593 | catalog=LIB1 | aflowlib_version=3.2.2d | aflowlib_date=20200407_13:35:32_GMT-5